plugins/otel-collector/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM golang:1.18-alpine3.14 AS otelcol
+
+RUN apk add --update --no-cache gcc g++
+ARG OTEL_COLLECTOR_VERSION=0.60.0
```

Comment:
shouldn't `OTEL_COLLECTOR_VERSION` be empty and get it only from build-args or build-args overwrite it?

Reply:
I think the arg there is a default value when it isn't specified by the build command. Do you prefer no default? It's also a good method to force a failure when there is no argument in the build command to make sure you have the argument in the build command. :)

Reply:
I don't mind to have a default.